docs/installation/installation-using-docker-application-setup.md

```diff
@@ -3,15 +3,58 @@
 This guide expects that you have already set up your Docker environment.
 If you haven't already done that check the [Installation Using Docker](installation-using-docker.md).
 
-## 1. Setup the application
-Now that the Docker environment is prepared we can setup the application itself.
+## 1. Setup the microservice for product search
+Installing of Shopsys Framework from a project-base requires manual cloning of the [repository of the microservice](https://github.com/shopsys/microservice-product-search).
+Clone the repository inside a separate directory and configure the container in your `docker-compose.yml`:
+```yaml
```

Comment:
Why is this part of documentation when it is already set in .dist docker-compose files? I would expect this to be maybe part of the UPGRADE.md so people know how to upgrade.

Reply:
I decided to let the developers know what exactly changed in the docker-compose. Do you think that this should be removed? If yes, I will remove it.

Reply:
I will remove it.